src/translations/screens/subscreens/TravelAidSettingsTexts.ts

```diff
@@ -0,0 +1,69 @@
+  header: {
+    accessibility: {
+      title: _('Tilgjengelighet', 'Accessibility', 'Tilgjengelegheit'),
+    },
+    howTo: {
+      title: _('Viktig å vite', 'Important information', 'Viktig å vite'),
+    },
+  },
+
+  toggle: {
+    title: _('Reisehjelp', 'Journey Aid', 'Reisehjelp'),
+    subText: _(
+      'Reisehjelp er en forenklet modus spesielt tilrettelagt for deg som trenger ekstra støtte når du reiser.\n\n* Stopp en buss eller trikk med appen, fra holdeplassen der du skal på.\n* Raskere tilgang på sanntids-oppdateringer om ankomst og neste holdeplass.\n* Snarvei til gyldige billetter.',
+      'Journey Aid is a simplified mode designed specifically for those who need extra support when traveling.\n\n* Use the app to signal a stop for a bus or tram from your boarding stop.\n* Get faster access to real-time updates on arrivals and upcoming stops, as well as a shortcut to your tickets.* Shortcut to valid tickets.',
+      'Reisehjelp er ein forenkla modus spesielt tilrettelagt for deg som treng ekstra støtte når du reiser.\n\n* Stopp ein buss eller trikk med appen, frå haldeplassen der du skal på.\n* Raskare tilgang på sanntids-oppdateringar om ankomst og neste haldeplass.\n* Snarveg til gyldige billettar.',
+    ),
+  },
+
+  button: {
+    importantInfo: {
+      title: _('Viktig å vite', 'Important information', 'Viktig å vite'),
+      a11yHint: _(
+        'Aktivér for å vise informasjonsside',
+        'Activate to show information page',
+        'Aktivér for å vise informasjonsside',
+      ),
+    },
+    contact: {
+      title: _(
+        'Ring kundeservice',
+        'Call Customer Service',
+        'Ring kundeservice',
+      ),
+      a11yHint: _(
+        'Aktivér for å ringe kundeservice',
+        'Activate to call customer service',
+        'Trykk for å ringe kundeservice',
+      ),
+    },
+  },
+
+  information: {
+    howTo: {
+      title: _(
+        'Hvordan sende stopp',
+        'How to send a stop',
+        'Hvordan sende stopp',
+      ),
+      content: _(
+        'Stoppsignal fra appen varsler sjåfør om at noen med ekstra behov skal av eller på.\n\n* I “Avganger”-fanen, velg holdeplassen du skal reise fra.\n* Fra buss-listen, velg riktig buss/trikk.\n* I visningen for bussen/trikken, trykk på Reisehjelp.\n* I Reisehjelp-modusen, trykk “Send stopp”.',
+        'A stop signal from the app alerts the driver that someone with extra needs wants to get off or on.\n\n* In the "Departures" tab, select the stop you are departing from.\n* From the bus list, choose the correct bus/tram.\n* In the view for the bus/tram, tap on "Travel Assistance."\n* In Travel Assistance mode, press "Send Stop."',
+        'Stoppsignal frå appen varslar sjåføren om at nokon med ekstra behov skal av eller på.\n\n* I "Avganger"-fanen, vel haldeplassen du skal reise frå.\n* Frå busslista, vel rett buss/trikk.\n* I visninga for bussen/trikken, trykk på "Reisehjelp."\n* I Reisehjelp-modusen, trykk "Send stopp."',
+      ),
+    },
```

Comment:
Same here regarding texts in app should be in sync with features in the app.

Reply:
Is not to be shown for this version.